release_notes.md

```diff
@@ -8,6 +8,7 @@
 - Updated PowerShell Worker (PS7) to [3.0.833](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.833)
 - Add IDistributedLockManager for Kubernetes environment (#7327)
 - Call EnableDrainModeAsync on ApplicationStopping (#7262)
+- Added Roslyn analyzer framework and avoid async void analyzer (#5612)
```

Comment:
nit: Added Roslyn analyzer projects and avoid async void analyzer

Reply:
Done.